app/Filament/Server/Pages/Settings.php

```diff
@@ -35,19 +44,181 @@ public function form(Schema $schema): Schema
                     ->columnSpanFull()
                     ->columns([
                         'default' => 1,
-                        'sm' => 2,
+                        'sm' => 1,
                         'md' => 4,
-                        'lg' => 6,
+                        'lg' => 8,
                     ])
                     ->schema([
+                        Fieldset::make()
```

Comment:
As discussed on discord, consolidate the "Icon" and "Information" fieldsets.

Like this, but without the big gap.
![Image](https://github.com/user-attachments/assets/ecfa171f-f0f5-4f60-913e-f9fbda061508)